test/integration/user_terms_seen_test.rb

```diff
@@ -25,12 +25,12 @@ def test_terms_presented_at_login
     assert_template "sessions/new"
     post "/login", :params => { :username => user.email, :password => "test", :referer => "/diary/new" }
     # but now we need to look at the terms
-    assert_redirected_to :controller => :users, :action => :terms, :referer => "/diary/new"
+    assert_redirected_to "/account/terms?#{{ :referer => '/diary/new' }.to_query}"
```

Comment:
```ruby
assert_redirected_to account_terms_path(:referer => "/diary/new")
```
... is a bit easier to read

Reply:
The entire test didn't use path helpers and has path expressions like `"/api/#{Settings.api_version}/user/preferences"`. I wrote `"/account/terms?#{{ :referer => '/diary/new' }.to_query}"` in case that was on purpose.

Updated to `account_terms_path(:referer => "/diary/new")`.